Added debug verbosity (#47)
redhat-cop/automation-good-practices: playbooks/README.adoc

----

====

== Use the verbosity parameter with debug statements
[%collapsible]
====
Explanations:: Debug messages should have a verbosity defined as appropriate for the message.

> Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production, but can cause concerns as they are shared between teams.

Reviewer: I would rather say that it clutters your log with possibly a lot of needless information which might confuse users.

Examples::
+
.Adding verbosity to debug messages
[source, yaml]
----

> - debug:
>    msg: "Don't make messages always display"

Reviewer: ```suggestion
- name: don't make messages always display
  debug:
    msg: "This message will clutter your log in production"
```
The indentation was slightly off and we recommend to always have a name

> - debug:
>    msg: "Show this only when verbosity is 2 or more"
>    verbosity: 2

Reviewer: ```suggestion
- name: this message will only appear when verbosity is 2 or more
  debug:
    msg: "Some more debug information if needed"
    verbosity: 2
```

----

====

== Use the verbosity parameter with debug statements
[%collapsible]
====
Explanations:: Debug messages should have a verbosity defined as appropriate for the message.

> Rationale:: Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production for future troubleshooting. However, log messages will clutter your output, which can confuse users with non-relevant information.

Reviewer 2: ```suggestion
Rationale::
Debug messages are useful during testing and development, and can be useful to retain as playbooks go into production for future troubleshooting.
However, log messages will clutter your output, which can confuse users with non-relevant information.
```
